apps/etterlatte-brev-api/src/main/kotlin/no/nav/etterlatte/brev/db/BrevRepository.kt

```diff
@@ -158,6 +159,29 @@ class BrevRepository(private val ds: DataSource) {
         }
     }
 
+    fun lagrePdf(
```

Comment:
Kan du no også slette metoden `lagrePdfOgFerdigstillBrev`?

Reply:
Ja og nja... Den brukes av "andre brev" (alt som ikke er vedtaksbrev). 
Vil nok skje noen endringer der på sikt, men vil helst unngå å ta det i denne PR-en siden det vil medføre at grensesnittet og flyten for opprettelse av andre brev må endres.